web/templates/left_sidebar.hbs

```diff
@@ -149,6 +149,12 @@
         <a class="zoom-out-hide" id="hide-more-direct-messages">
             <span> {{t 'back to channels' }}</span>
         </a>
+        <div class="input-append zoom-out-hide direct-messages-search-section">
+            <input class="direct-messages-list-filter filter_text_input" type="text" autocomplete="off" placeholder="{{t 'Filter direct messages' }}" />
+            <button type="button" class="btn clear_search_button" id="clear_direct_messages_search_button">
```

Comment:
We're trying to [hyphenate new and rewritten classes and IDs](https://zulip.readthedocs.io/en/latest/subsystems/html-css.html#use-clear-unique-names-for-classes-and-object-ids), so `clear-direct-messages-search-button` would be preferable here.